pyproject.toml

```diff
@@ -52,11 +52,13 @@ pdbpp = "^0.10.3"
 pytest = "^6.2.5"
 pytest-cov = "^3.0.0"
 pytest-env = "^0.6.2"
+a3b2bbc3ced97675ac3a71df45f55ba = "^6.4.0"
```

Comment:
What is this?

Reply:
`lhapdf` in disguise

Reply:
https://pypi.org/project/a3b2bbc3ced97675ac3a71df45f55ba/

Reply:
Unfortunately, it is not working on Mac M1 (not yet), and Andy Buckley did not accept yet my [Merge Request](https://gitlab.com/hepcedar/lhapdf/-/merge_requests/16), so it's still unofficial...

Reply:
it looks he forgot about it..